wallet/lib/utils/extension.dart

```diff
@@ -36,6 +36,13 @@ extension ConvertToUSD on String {
     }
     return '';
   }
+  String convertPylonsToUSD(String amount) {
+    switch (this) {
```

Comment:
Why we are using switch here?

Reply:
The 1166 branch already has a function for converting pylon to usd. So I copied the same code from there to avoid any future conflicts.

Reply:
Switch are used for options that are more than 2 not for exact two. Kindly change it.